examples/text_classification/multi_label/train.py

```diff
@@ -93,12 +96,7 @@ def do_train():
     train_ds = load_dataset(read_custom_data, filename=os.path.join(
         args.data_path, file_name), is_test=False, lazy=False)
 
-    # If you wanna use ernie pretrained model,
```

Comment:
@songzy12 Thanks for your contribution! Since the example of multi-label text classification follows the strategy of fine-tuning from the pretrained model. This comment here tells the users that they can also use the other pretrained models provided by PaddleNLP Transformer API.

Reply:
Thanks for the review!

Regarding the pretrained model example, I plan to use an extra flag to select which model  to use like in the rnn example [1] rather than keep them in the comment [2]. Does this plan seems good?

[1] https://github.com/PaddlePaddle/PaddleNLP/blob/af82e03bbfc50ee4748a90f914c347f1e3f6ddcc/examples/text_classification/rnn/train.py#L104-L159
[2] https://github.com/PaddlePaddle/PaddleNLP/blob/af82e03bbfc50ee4748a90f914c347f1e3f6ddcc/examples/text_classification/pretrained_models/train.py#L151-L164

Reply:
Thanks a lot for the advise! 

For Simplicity, the pretrained model is default to BERT in this example. 
And we have updated the usage instruction of pretrained models in multi-label text classification: https://github.com/PaddlePaddle/PaddleNLP/pull/979